README.md

```diff
@@ -129,7 +131,8 @@ export WSK_PORT=$(kubectl -n openwhisk describe service nginx | grep https-api |
 Now you should be able to setup the wsk cli like normal and interact with Openwhisk.
 
 ```
-wsk property set --auth $AUTH_SECRET --apihost https://[kube_node_ip]:$WSK_PORT
+export KUBE_IP=$(kubectl get nodes | grep Ready | awk '{ print $1;exit }')
```

Comment:
Would it make more sense to set this on line [123](https://github.com/IBM/OpenWhisk-on-Kubernetes/pull/11/files#diff-04c6e90faac2675aa89e2176d2eec7d8R123)?